mkdocs/contrib/search/__init__.py

```diff
@@ -40,7 +40,7 @@ class SearchPlugin(BasePlugin):
     config_scheme = (
         ('lang', LangOption(default=['en'])),
         ('separator', config_options.Type(utils.string_types, default=r'[\s\-]+')),
-        ('prebuild_index', config_options.Type(bool, default=False)),
+        ('prebuild_index', config_options.Type(utils.string_types, default='')),
```

Comment:
I understand the change here, but what about users who have `True` or `False` already set in their config? This will error as an invalid type. Seems to me `False` should be treated the same as an empty string and `True` should be treated as `'node'`.